xwiki-rendering-api/src/main/java/org/xwiki/rendering/block/AbstractBlock.java

```diff
@@ -399,6 +404,40 @@ public void setParameters(Map<String, String> parameters)
         }
     }
 
+    @Override
+    public Map<String, Object> getAttributes()
+    {
+        return this.attributes == null ? Collections.emptyMap()
```

Comment:
I'm wondering is there is some meaningful differences with `Map.of()/Map.copyOf`.

Reply:
I don't think so, to be honest I've just copied the code for parameters and change the names.